projects/plugins/jetpack/modules/custom-css/custom-css/preprocessors/lessc.inc.php

```diff
@@ -2404,6 +2415,8 @@ protected function parseChunk() {
 		if (empty($this->buffer)) return false;
 		$s = $this->seek();
 
+		$block = null;
```

Comment:
Rector is probably not realizing that line 2518 will always throw. If we have to do anything, it would probably make more sense to put the dummy assignment after that line inside the catch block.

A comment there pointing out that the dummy assignment is just to make rector happy wouldn't be amiss either.